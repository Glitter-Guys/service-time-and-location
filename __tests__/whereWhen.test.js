import React from 'react';
import expect from 'expect';
import WhereWhen from './../client/src/components/whereWhen.jsx';

describe('<WhereWhen />', () => {
 it('should be defined', () => {
   expect(WhereWhen).toBeDefined();
 });
 
 it('should render', () => {
   const WhereWhenWrap = <WhereWhen />;
   expect(WhereWhenWrap).toMatchSnapshot();
 })
});
