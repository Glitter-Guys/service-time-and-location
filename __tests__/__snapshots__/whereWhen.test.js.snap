// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WhereWhen /> should render 1`] = `<WhereWhen />`;
